iiop-openjdk/src/main/resources/schema/jboss-as-iiop-openjdk_1_0.xsd

```diff
@@ -46,15 +46,16 @@
             </xs:documentation>
         </xs:annotation>
         <xs:sequence>
+            <xs:element name="properties" minOccurs="0" maxOccurs="1" type="genericPropertiesType"/>
```

Comment:
If I am reading https://github.com/wildfly/wildfly/blob/master/iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/IIOPRootDefinition.java#L333 correctly, I think this one comes just before the transport-config element?